docs/tech/vumi.md

```diff
@@ -0,0 +1,25 @@
+# Vumi
+
+We build on top of our Vumi platform for SMS, USSD and other messaging protocols.
+
+[Vumi][] is a scalable messaging engine which we use for SMS, USSD and other messaging
+protocols.
+
+We [build Vumi in a container][docker-vumi] and [push it to Docker Hub][dockerhub].
+
+[Vumi Go][] is a hosted version of Vumi. Where Vumi gives you the tools to
+build large scale messaging applications, Vumi Go provides you with a working
+environment that is already integrated into numerous countries.
```

Comment:
Not sure if you want to mention that Vumi Go is deprecated and going away by the end of the year.